src/components/Menus/Sidebar/DownloadPanel/DownloadPanel.jsx

```diff
@@ -12,6 +13,7 @@ import "../../../../assets/stylesheets/DownloadPanel.scss";
 
 export const DownloadPanel = () => {
   const { t } = useTranslation();
+  const user = useSelector((state) => state?.auth?.user);
```

Comment:
We've never needed the `?`s before 🤔 Wondering whether this is coming from the tests, and if so I think we've been representing a non-logged in user with `auth: {}` (so this shouldn't fail if there's no user)

Reply:
We use optional chaining in the `ProjectsPanel` and the `ProjectsIndex` but I've removed it here for now and passed in an empty `auth` for the tests.

Is there a reason not to use optional chaining? Could it be a useful guard just in case the auth or the user is missing?

Anyway, that might be a discussion for another time so I've removed for now!